### DISCUSSION TITLE
Where is the option to close the instance.

### DISCUSSION THREAD

**Discussion by SUPTECGourry** [misskey-dev/misskey#13593] [Q&A] [answered]
When i first setup my instance i set as closed. few days ago i found some users using my instance, when i went to check the settings, the option to set the instance closed disappeared. did the option moved to another place? (v2024.3.1)

![image](https://github.com/misskey-dev/misskey/assets/139704542/3f4e449d-3e53-47db-b69a-700300555169)

**Comment by Hoto-Cocoa** [comment, marked as answer]
If you are saying about close registration, check the "Moderation" tab.
